modin/pandas/test/dataframe/test_map_metadata.py

```diff
@@ -1089,9 +1089,8 @@ def test_insert_dtypes(data, astype):
 @pytest.mark.parametrize("loc", int_arg_values, ids=arg_keys("loc", int_arg_keys))
 def test_insert_loc(data, loc):
     modin_df, pandas_df = pd.DataFrame(data), pandas.DataFrame(data)
-    value = modin_df.iloc[:, 0]
 
-    eval_insert(modin_df, pandas_df, loc=loc, value=value)
+    eval_insert(modin_df, pandas_df, loc=loc, value=lambda df: df.iloc[:, 0])
```

Comment:
Why make this change?

Reply:
I have no idea why this _worked_ before that change - we were effectively telling pandas to insert a Modin Series, and for some reason it worked in <1.5, but now broke.
I don't think we ever intended to support such cross-operation, so I decided to fix the test instead of trying to find out why it started to fail in 1.5.

Reply:
You're testing inserting the callable itself. Maybe we could do something like `eval_general(modin_df, pandas_df, operation=lambda df: df.insert(loc=loc, column="new column", value=df.iloc[:, 0]), __inplace__=True)`

Reply:
No, I'm not 😄 

`eval_insert()` calls `eval_general()` under the hood, and `eval_general()` transforms callables by calling them against both pandas and Modin dataframes before actually calling the function to evaluate:
https://github.com/modin-project/modin/blob/94ca77104ed296e51ddb1cc6b8880fe29babc11c/modin/pandas/test/utils.py#L746-L762

Reply:
Oh, alright!